src/SDK/SmartDetectorsSDK/AlertPresentation/ThresholdType.cs

```diff
@@ -0,0 +1,24 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="ThresholdType.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace Microsoft.Azure.Monitoring.SmartDetectors.AlertPresentation
+{
+    /// <summary>
+    /// Enumeration of possible types of a metric chart threshold.
+    /// </summary>
+    public enum ThresholdType
```

Comment:
>ThresholdType [](start = 16, length = 13)

There should also be "Both"

Reply:
when it's added to the contracts, i'll add it here

---
In reply to: [258925051](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/52#discussion_r258925051) [](ancestors = 258925051)